src/enhanced-textarea.jsx

```diff
@@ -73,8 +77,17 @@ const EnhancedTextarea = React.createClass({
     };
   },
 
+  handleResize(e) {
+    this._syncHeightWithShadow(undefined, e);
+  },
+
   componentDidMount() {
     this._syncHeightWithShadow();
+    window.addEventListener('resize', this.handleResize);
+  },
+
+  componentWillUnmount: function() {
+    window.removeEventListener('resize', this.handleResize);
```

Comment:
But we should probably open another PR for that. To both discuss and demonstrate that library.